### PR TITLE
fix verify you phone page

### PR DESCRIPTION
**Pull request by FarahJamal-business**
### Jira URL

https://optimumpartners.atlassian.net/browse/REV-140

### CODE REVIEW

File: src/components/verify-phone/VerifyPhoneForm/index.jsx [hunk 1]
```diff
@@ -113,7 +113,7 @@ function VerifyPhoneForm() {
           </CountdownCircleTimer>
 
           <ResendButton onClick={() => setResendCode(!resendCode)}>
-            Did not receive code? send again
+            Didn&apos;t receive a code? Send it again
```
Comment (mohdabed123) [inline src/components/verify-phone/VerifyPhoneForm/index.jsx:116]: &apos isn not a valid html character if we want to use an apostrophe use it like this
```suggestion
           { "Didn&apos;t receive a code? Send it again"}
```

please check if there anywhere else that uses &apos and fix it as suggested.

Comment (FarahJamal-business) [reply src/components/verify-phone/VerifyPhoneForm/index.jsx:116]: sorry that didn't work!

Comment (mohdabed123) [reply src/components/verify-phone/VerifyPhoneForm/index.jsx:116]: try this  {`
            Didn't receive a code? Send it again`}

File: src/components/verify-phone/VerifyPhoneForm/index.jsx [hunk 2]
```diff
@@ -25,6 +25,7 @@ import {
  */
 function VerifyPhoneForm() {
   const [resendCode, setResendCode] = useState(true);
+  const message = `Didn't receive a code? Send it again`;
```
Comment (mohdabed123) [inline src/components/verify-phone/VerifyPhoneForm/index.jsx:28]: +1

**Review by mohdabed123 — APPROVED** [review]

**Pull request merged**